src/backend/aspen/api/middleware/session.py

```diff
@@ -0,0 +1,85 @@
+import hashlib
+
+from flask.json.tag import TaggedJSONSerializer
+from itsdangerous import URLSafeTimedSerializer
+from itsdangerous.exc import BadSignature
+from starlette.datastructures import MutableHeaders
+from starlette.requests import HTTPConnection
+from starlette.types import ASGIApp, Message, Receive, Scope, Send
+
+
+class SessionMiddleware:
+    def __init__(
+        self,
+        app: ASGIApp,
+        secret_key: str,
+        session_cookie: str = "session",
```

Comment:
is this supposed to be something more intense/ not easily guessable?

Reply:
Nah, it's just the name of the cookie. It's supposed to match the name of the cookie that the flask app uses (which is just "session" by default)